src/components/Dropdown/DropdownTrigger.react.js

```diff
@@ -10,6 +10,7 @@ type Props = {|
   +toggle?: boolean,
   +value?: string,
   +type?: "link" | "button",
+  +color?: string,
```

Comment:
Should we make this a string enum i.e. `'red' | 'blue' | 'orange'` etc?

Reply:
@AaronCoplan I was going to ask about this, as we re-use the same list of colors in multiple places, is it worth having a separate color type somewhere that we can use for them all?